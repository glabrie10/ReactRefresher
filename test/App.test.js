import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';
import { testScheduler } from 'jest';
import RelatedItems from '../client/src/components/App.jsx';
configure({ adapter: new Adapter() });
test('rendering the app component', () => {
  const wrapper = shallow(
    <RelatedItems />,
  );
  expect(wrapper).toMatchSnapshot();
});

