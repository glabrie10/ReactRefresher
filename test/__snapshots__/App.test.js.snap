// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering the app component 1`] = `ShallowWrapper {}`;
